site/en/user_guide.md

```diff
@@ -214,19 +229,22 @@ docker run --rm -u $(id -u) -v $PWD:/data -w /data -- ocrd/all:maximum ocrd-[pro
 ```
 Your command could e.g. look like this:
 ```sh
-ocrd-tesserocr-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-TESSEROCR -P model Fraktur
+ocrd-olena-binarize -I OCR-D-IMG -O OCR-D-BIN -P impl sauvola
 ## alternatively using docker
-docker run --rm -u $(id -u) -v $PWD:/data -w /data -- ocrd/all:maximum ocrd-tesserocr-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-TESSEROCR -P model Fraktur
+docker run --rm -u $(id -u) -v $PWD:/data -w /data -- ocrd/all:maximum ocrd-olena-binarize -I OCR-D-IMG -O OCR-D-BIN -P impl sauvola
 ```
 
+The specified processor will take the files in your Input-Group -I, process them and save the results in your Ouput-Group -O. It will also add
+the information about this processing step and its results to METS file in your workspace. 
+
 **Note:** For processors using multiple input-, or output groups you have to use a comma separated list. 
 
 E.g.: 
 
 ```sh
-ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-BIN,OCR-D-IMG-BIN
+ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-CROP,OCR-D-IMG-CROP
```

Comment:
No 2 output filegroups anymore since 0.0.5:

```
ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-CROP
```

Reply:
oh, my version still produces to output filegroups. same for `cis-ocropy-denoise` and `cis-ocropy-deskew`. To which processors does this still apply then?

Reply:
ocrd-anybaseocr: you're right, I opened a PR https://github.com/OCR-D/ocrd_anybaseocr/pull/66 but did not finish it. thx.

in ocrd_cis, merging https://github.com/cisocrgroup/ocrd_cis/pull/57 will change the behavior.

Let's keep the two-output-filegrps convention in the docs, I hope I'll get a new release of ocrd_all ready this week with all the processor fixes, then we can adapt it and I can properly answer your question.